### ISSUE TITLE
Tabs vs. spaces?

### ISSUE THREAD

**Issue by ajxu2** [cpinitiative/ide#56]
As far as I can tell, the IDE always indents with tabs rather than spaces. It would be nice if there was a setting to change whether the IDE uses tabs or spaces and also a setting to adjust indent size.

Thanks!

**Comment by bqi343** [comment]
As mentioned in #61, Indent size is already supported, but tabs vs spaces hasn't been supported yet because changing it in the code editor also changes it in the input and scribble editors, which seems undesirable. Though maybe it doesn't really matter (how often do you need to type tabs in the input anyways?)

**Comment by thecodingwizard** [comment]
yeah, I think it's worth adding tabs vs spaces option (along with a warning that it will also impact tabs in the input field)
